code/game/objects/structures/mirror.dm

```diff
@@ -1,3 +1,20 @@
+
+// Normal Mirrors
+
+#define CHANGE_HAIR "Change Hair"
+#define CHANGE_BEARD "Change Beard"
+
+// Magic Mirrors!
+
+#define CHANGE_RACE "Change Race"
+#define CHANGE_SEX  "Change Sex"
+#define CHANGE_NAME "Change Name"
+#define CHANGE_EYES "Change Eyes"
+
+#define INERT_MIRROR_OPTIONS list(CHANGE_HAIR, CHANGE_BEARD)
+#define PRIDE_MIRROR_OPTIONS list(CHANGE_HAIR, CHANGE_BEARD, CHANGE_RACE, CHANGE_SEX, CHANGE_EYES)
+#define MAGIC_MIRROR_OPTIONS list(CHANGE_HAIR, CHANGE_BEARD, CHANGE_RACE, CHANGE_SEX, CHANGE_EYES, CHANGE_NAME)
```

Comment:
these all need to be `#undef`'d at the end of your file. we typically have a CI check for this but it's apparently broken but this still needs to be done